Lesson_35/js/script.js

```diff
@@ -0,0 +1,32 @@
+$(document).ready(function () {
```

Comment:
тут можно использовать shorthand
```suggestion
$(function() {
```